parts/language/lang_vi.lua

```diff
@@ -990,7 +990,7 @@ return {
         "It’s possible to finish 40L without rotation buttons.",
         "Join our Discord!",
         "l-=-1",
-        "Let the bass kick!",
+        "Biết ông Thương không?",
         "Low frame rates reduce your gaming experience.",
         "LrL RlR LLr RRl RRR LLL FFF RfR RRf rFF",
```

Comment:
This is a guide to solve a rubik. If I remember correctly, it's name is "||Sexy|| dance". (I'm not joking)